### PR TITLE
chronos lib test to improve coverage

### PR DESCRIPTION
**Pull request by gganapavarapu**
Chronos lib test to improve coverage.

### CODE REVIEW

File: services/inference/tests/test_inference_lib.py [hunk 1]
```diff
@@ -84,6 +85,7 @@ def _basic_result_checks(results: PredictOutput, df: pd.DataFrame):
     assert results["date"].iloc[-1] - df["date"].iloc[-1] == timedelta(hours=FORECAST_LENGTH)
 
 
+@pytest.mark.parametrize("forecasting_input_base", [TTM_MODEL_ID], indirect=True)
```
Comment (wgifford) [inline services/inference/tests/test_inference_lib.py:88]: Can we use this same test with "chronos"?

File: services/inference/tests/test_inference_lib.py [hunk 2]
```diff
@@ -94,6 +96,7 @@ def test_forecast_with_good_data(ts_data_base: pd.DataFrame, forecasting_input_b
     _basic_result_checks(results, df)
 
 
+@pytest.mark.parametrize("forecasting_input_base", [TTM_MODEL_ID], indirect=True)
```
Comment (wgifford) [inline services/inference/tests/test_inference_lib.py:99]: Do we need to parameterize if there is only one parameter? Or can we also add chronos here?

Comment (gganapavarapu) [reply services/inference/tests/test_inference_lib.py:99]: assert for chronos is different. The resulting dataframe does not have date column. So, chronos should be in a separate test function. If any preprocessing and post processing of data is enabled for chronos, we will be able to use the same test.

File: services/inference/tests/test_inference_lib.py (hunk 1 above)
Comment (gganapavarapu) [reply services/inference/tests/test_inference_lib.py:88]: assert for chronos is different. We can do asserts based on model. Separate methods would help if we use different parameters for both the models. We can even expand the `forecasting_input_base` parameters to consider model parameters.

File: services/inference/tests/test_inference_lib.py (hunk 2 above)
Comment (wgifford) [reply services/inference/tests/test_inference_lib.py:99]: We need the result to contain a date column

Comment (wgifford) [reply services/inference/tests/test_inference_lib.py:99]: Chronos vs. TTM should produce a similarly formatted result, and take all the same inputs.

File: services/inference/tests/test_inference_lib.py (hunk 1 above)
Comment (gganapavarapu) [reply services/inference/tests/test_inference_lib.py]: Extended all the TTM tests to Chronos.

File: services/inference/tests/test_inference_lib.py (hunk 2 above)
Comment (gganapavarapu) [reply services/inference/tests/test_inference_lib.py]: Added date column in the output. Updated the chronos handler to use TimeSeriesPreprocessor to handle issues around empty target column list, frequency etc. This allowed to extend the TTM units to Chronos.

**Pull request merged**